spec/features/create_denver_book_chapter_spec.rb

```diff
@@ -41,18 +41,14 @@
   end
 
   it 'applys work visibility' do
-    visit "concern/#{work_type.to_s.pluralize}/new"
     visibility = :open
```

Comment:
Don't forget to remove these now as they are redundant.

Reply:
Change this line to choose("#{work_type}_visibility_open}") ??

Reply:
`choose("#{work_type}_visibility_#{visibility}")` could just be `choose("#{work_type}_visibility_open")` and we can get rid of that variable.